lessons/Data-Science/histograms-interpret/langs/en-us/index.adoc

```diff
@@ -0,0 +1,323 @@
+= Interpreting the "Shape" of Data
+
+@description{Students explore how their understanding of histogram "shape" can help them to interpret data.}
+
+@lesson-prereqs{histograms-visualize}
+
+@keywords{histogram, bin, interval}
+
+[@lesson-intro-table]
+|===
+| Lesson Goals
+| Students will be able to...
+
+@objectives
+
+| Student-facing Lesson Goals
+|
+
+- Let's investigate what the shape of a histogram can tell us about the data.
+
+| Materials
+|[.materials-links]
+
+@material-links
+
+|===
+
+
+== Interpreting Shape
+
+@objective{histogram-shape}
+
+=== Overview
+
+Students consider what different distributions tell us about a dataset, connecting different scenarios with different data visualizations.
+
+=== Launch
+
+We now have the terminology needed to describe histogram shape. But what can we actually conclude about a dataset based on a histogram's shape?
+
+@lesson-instruction{
+
+Let's consider two different scenarios:
+
+- The average US woman gives birth around age 26, but some do even after 45! Children do not give birth.
+
+- Most adults have close to a full set of 32 teeth, but a few hockey players might have a very small number of teeth. No one has more than 32 teeth.
+
+*Let's draw a rough sketch of what we think the histogram for each of these scenarios might look like.*}
+
+@teacher{
+Invite a student to the board to draw a rough sketch of each scenario. Invite the students to describe how they decided where to draw the clusters, peaks, and outliers.}
+
+@QandA{
+@Q{Which of the above scenarios describes a left skew? Explain.}
+@A{The scenario about women's ages describes a left skew. Most of the data is clustered around 26, but there is also data further on the right, all the way up to 45. There is no data around, say, age 7.}
+
+@Q{Which of the above scenarios describes a right skew? Explain.}
+@A{The scenario about teeth describes a right skew. Most of the data is clustered around 32 with a few outliers further to the left.}
+}
+
+@lesson-instruction{
+- With a partner, complete @printable-exercise{use-shape-to-interpret.adoc}.
+- Just like we did during the warm up, you will sketch rough histograms, make a decision their shape, and then interpret the data.
+}
+
+=== Investigate
+
+@teacher{
```

Comment:
@retabak the materials table of this lesson definitely needs a preparation section!

Reply:
Although I'm starting to wonder whether this chart paper activity makes more sense on printed pieces of paper 

- we could make the table with the headers  
- they could work in their seats
- there could still be a gallery walk
- teachers wouldn't have to store giant chart paper until the box plots lesson
- they could fit on a bulletin board

Reply:
I dunno.... it's nice to save prep, but I like it as is. (Also, Peter Liljedahl advocates for kids doing work standing up at the board as often as possbile. I could add a teacher note with a citation, if you'd like.) On a small piece of paper, it's harder for everyone to see/ discuss/ contribute. The directions (rotate to the next paper) somehow feel more intuitive, too.

If you feel strongly, let me know and I'm happy to make a worksheet. I prefer it how it is, but it's not terribly important to me either way. @joystraub or @schanzer , who have facilitated this activity during PD, may want to weigh in.

Reply:
Chiming in here - I agree that this is much better off done with students standing.

Reply:
I love butts out of seats activities... but I hate piles of chart paper hanging around my miniature classroom that I share with other teachers and 120 students inhabit every day... 

Unlike in PD, they won't get to box plots for several days... and since @retabak hasn't actually added the corresponding instructions to the box plot lesson yet, I can't tell whether it might actually be something that'll be kicking around the classroom for over a week. In my world, where I shared rooms with other teachers, that might mean it wasn't in a usable state anymore.

@schanzer @joystraub How important is it for them to do the box plots activity using this same chart paper?

Reply:
I think that @schanzer and @joystraub both think it is important to work at the wall on large paper. I'll let them weigh in about the importance of doing box plots on the same paper.

I'd rather give instructions for a more impactful activity than a more convenient activity. I think teachers who are in your shoes may need to modify accordingly, if we are going with the majority.  Will add a "Preparation" section to the lesson table unless you want to continue the debate, @flannery-denny. LMK.

Reply:
please add a preparation section @retabak